components/centraldashboard/public/components/default-notebooks-card.js

```diff
@@ -0,0 +1,104 @@
+import '@polymer/iron-ajax/iron-ajax.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/iron-icons/iron-icons.js';
+import '@polymer/paper-card/paper-card.js';
+import '@polymer/paper-item/paper-icon-item.js';
+import '@polymer/paper-ripple/paper-ripple.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/paper-progress/paper-progress.js';
+import './iframe-link.js';
+// eslint-disable-next-line max-len
+import {AppLocalizeBehavior} from '@polymer/app-localize-behavior/app-localize-behavior.js';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+
+import {html, PolymerElement} from '@polymer/polymer';
+
+import './card-styles.js';
+
+/**
+ * Component to retrieve and display recently modified Jupyter Notebooks.
+ */
+// eslint-disable-next-line max-len
+export class FirstNotebookCard extends mixinBehaviors([AppLocalizeBehavior], PolymerElement) {
+    static get template() {
+        return html`
+        <style include="card-styles">
+            :host {
+                @apply --layout-vertical;
+            }
+        </style>
+        <iron-ajax auto url="[[defaultNotebookUrl]]" handle-as="json"
+            loading="{{loading}}" on-response="_onNotebookServersResponse"
+            on-error="_onError">
+        </iron-ajax>
+            <paper-card heading="Default Notebook">
+            <table>
```

Comment:
This doesn't seem to match exactly with the mockup that I remember seeing here https://github.com/StatCan/aaw/issues/1911#issuecomment-1883347754.

You added the name (but that is a good change that I agree with). But there isn't the status and the type for the notebook, and here you say "Type" but then display the image.
Was that just because you missed those fields or did the design change?